api-catalog-ui/frontend/src/components/Wizard/WizardDialog.jsx

```diff
@@ -76,6 +80,130 @@ export default class WizardDialog extends Component {
         }
     };
 
+    /**
+     * Convert an uploaded yaml file to JSON and save in redux
+     */
+    showFile = (e) => {
+        e.preventDefault();
+        const reader = new FileReader();
+        const filepath = e.target.value.split('\\');
+        const filename = filepath[2];
+        reader.onload = (event) => {
+            const text = event.target.result;
+            try {
+                const obj = yaml.load(text);
+                this.fillInputs(obj);
+                this.props.updateUploadedYamlTitle(filename);
+            } catch {
+                document.getElementById('yaml-browser').value = null;
+                toast.warn('Please make sure the file you are uploading is in valid YAML format!', {
```

Comment:
It would be better to create an action for the warn message, similarly to how is done for the others messages.
Further, the message is not getting dismissed. You might fix the problem by using `setTimeout` like in the wizard-fetch-actions.jsx.